cmd/create/create_scheduler.go

```diff
@@ -0,0 +1,129 @@
+// maestro-cli
+// https://github.com/topfreegames/maestro-cli
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package create
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/spf13/cobra"
+	"github.com/topfreegames/maestro-cli/common"
+	"github.com/topfreegames/maestro-cli/extensions"
+	"github.com/topfreegames/maestro-cli/interfaces"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	v1 "github.com/topfreegames/maestro/pkg/api/v1"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var marshler = &runtime.HTTPBodyMarshaler{
+	Marshaler: &runtime.JSONPb{
+		MarshalOptions: protojson.MarshalOptions{
+			EmitUnpopulated: true,
+		},
+	},
+}
+
+// createSchedulerCmd represents the create command
+var createSchedulerCmd = &cobra.Command{
```

Comment:
A good thing would be adding some examples of usage (very simple).

A reference could be the kubectl. It defines the [description as a constant](https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/apply/apply.go#L119) and then [use on the cobra command](https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/apply/apply.go#L175).